pkg/filters/int.go

```diff
@@ -53,57 +67,89 @@ func (f *IntFilter) Enabled() bool {
 }
 
 func (f *IntFilter) Minimum() int64 {
-	return f.GreaterThan
+	return f.greaterThan
 }
 
 func (f *IntFilter) Maximum() int64 {
-	return f.LessThan
+	return f.lessThan
 }
 
-func (filter *IntFilter) Parse(operatorAndValues string) error {
-	if len(operatorAndValues) < 2 {
-		return fmt.Errorf("invalid operator and/or values given to filter: %s", operatorAndValues)
+// priority goes by (from most significant):
+// 1. equality
+// 2. greater
+// 3. lesser
+// 4. non equality
+func (f *IntFilter) Filter(val int64) bool {
+	result := !f.enabled || f.equal[val] || val > f.greaterThan || val < f.lessThan
+	if !result && f.notEqual[val] {
+		return false
 	}
-	valuesString := string(operatorAndValues[1:])
-	operatorString := string(operatorAndValues[0])
+	return result
+}
 
-	if operatorString == "!" {
-		if len(operatorAndValues) < 3 {
-			return fmt.Errorf("invalid operator and/or values given to filter: %s", operatorAndValues)
-		}
-		operatorString = operatorAndValues[0:2]
-		valuesString = operatorAndValues[2:]
+func (f *IntFilter) validate(val int64) bool {
+	const maxIntVal32Bit = math.MaxInt32
+	const minIntVal32Bit = math.MinInt32
```

Comment:
I don't see the point of redeclaring these consts

Reply:
Already addressed this comment, I mostly did it as a mnemonic for myself while writing the code. And technically if we ever wanted different values it would be a one time change here. 
I think it's more appropriate to just change the names as we've also discussed.

Reply:
Fixed